packages/devtools_app/test/shared/preferences_controller_test.dart

```diff
@@ -15,13 +15,18 @@ import '../test_infra/flutter_test_storage.dart';
 void main() {
   setGlobal(ServiceConnectionManager, FakeServiceManager());
 
-  group('PreferencesController', () {
+  group('$PreferencesController', () {
```

Comment:
why are we adding the `$` char in group names? This isn't done anywhere else in our test code, so let's remove to be consistent

Reply:
With $, if the class is renamed, the group name will catch up the update. Yes, as any innovation, it is inconsistent with previous pattern. 
You think the benefit does not worth the break of consistency?

Reply:
Ah, nice. Today I learned that you can use dart string interpolation on Classes. Good to leave.